frontend/src/components/data-table/data-table.tsx

```diff
@@ -112,6 +121,15 @@ const DataTableInternal = <TData,>({
     onColumnFiltersChange: onFiltersChange,
     // selection
     onRowSelectionChange: onRowSelectionChange,
+    initialState: {
+      columnPinning: {
+        left:
+          freezeColumnsLeft === undefined
+            ? ["__select__"]
```

Comment:
could we share this const `const SELECT_COLUMN_ID = "__select__"` somewhere?

Reply:
I put it in `types.ts`, let me know if there's a better spot